src/energyplus/ForwardTranslator/ForwardTranslateBlind.cpp

```diff
@@ -189,6 +195,21 @@ namespace energyplus {
       idfObject.setDouble(WindowMaterial_BlindFields::MaximumSlatAngle, *d);
     }
 
+    // Call the translation of these objects, which has two advantages:
+    // * will not translate them if they are orphaned (=not referencing a material), and,
+    // * makes the order of these objects in the IDF deterministic
+    if (boost::optional<MaterialPropertyMoisturePenetrationDepthSettings> _empd = modelObject.materialPropertyMoisturePenetrationDepthSettings()) {
+      translateAndMapModelObject(_empd.get());
+    }
+
+    if (boost::optional<MaterialPropertyPhaseChange> _phaseChange = modelObject.materialPropertyPhaseChange()) {
+      translateAndMapModelObject(_phaseChange.get());
+    }
+
+    if (boost::optional<MaterialPropertyPhaseChangeHysteresis> _phaseChangeHysteresis = modelObject.materialPropertyPhaseChangeHysteresis()) {
+      translateAndMapModelObject(_phaseChangeHysteresis.get());
+    }
```

Comment:
PCM not accepted for blind. 

https://github.com/NREL/EnergyPlus/blob/0c9b8c4944b5f06edb000bf0d766f067830af0ae/src/EnergyPlus/HeatBalanceManager.cc#L3421

Neither is MoisturePenetration, which also only accepted a RegularMaterial.

https://github.com/NREL/EnergyPlus/blob/0c9b8c4944b5f06edb000bf0d766f067830af0ae/src/EnergyPlus/MoistureBalanceEMPDManager.cc#L209-L229